md/ru/Changelog.md

```diff
@@ -22,7 +22,64 @@
 * Исправлено развертывание многоцелевых пакетов Debian
 * Фикс развертывания qt6 qml приложений
 
+## CQtDeployer 1.5.0
+
+### Нововведения
+
+- Добавлена новая тема установщика qif "quasarDark".
+- Добавлена поддержка работы с кастомным шаблоном для опции qif.
+- Добавлена поддержка пакетов debian.
+- Добавлены значки поддержки для целей. теперь опция значка работает с целями, но не с пакетами.
+- Добавлена поддержка управления файлами пользовательского перевода.
+- Добавлена поддержка развертывания неисполняемых данных. (опция extraData)
+- Добавлены переменные окружения для развернутых приложений.
+- Добавлена поддержка создания пакетов deb.
+- Добавлена поддержка создания zip-архивов.
+- Добавлена поддержка создания установщика qif с использованием настраиваемого шаблона.
+- Добавлена команда инициализации qif по умолчанию шаблона deb (getDefaulttemplate)
+- Добавлена поддержка импорта пользовательских скриптов запуска.
+- Добавлена поддержка хэшей md5 пакетов.
+- Добавлена поддержка Qt6.1.
+
+### Исправления
+
+- Исправлена стабильность для опции deploySystem в Windows.
+- Исправлен подробный журнал.
+- Исправлен общий журнал вывода.
+- Исправлены опечатки в логе.
+- Исправлено кросс-развертывание arm дистрибутива.
+- Исправлено развертывание Qt6 qml
+- Исправлены команды PowerShell в Windows.
+- исправления ошибок и улучшения.
+
+### Новые переменные среды
+
+- CQT_RUN_FILE - эта переменная содержит путь к сценарию выполнения используемого приложения.
+- CQT_PKG_ROOT - эта переменная содержит путь к корню текущего пакета.
+
+### Новые параметры
+
+- zip - создать ZIP-архив для развертываемых программ
+- deb - Создать пакет deb для программы развертывания
+- homepage - Устанавливает URL-адрес домашней страницы для пакета
+- noQt - игнорирует ошибку инициализации qmake. Используйте только в том случае, если ваше приложение не использует фреймворк qt
+- allowEmptyPackages - позволяет настраивать пустые пакеты.
+- getDefaultTemplate - Извлекает шаблоны deb или qif по умолчанию.
+- tr - Добавляет qm файлы в папку переводов.
+- extraData - добавляет дополнительные файлы или каталоги как цель. Выбранный каталог будет скопирован в папку extraDataOut с сохранением собственной структуры.
+- extraDataOut - Устанавливает путь к каталогу с дополнительными файлами данных.
+- prefix - Устанавливает префикс для пакета относительно целевого каталога
+- homePage - Устанавливает URL-адрес домашней страницы для пакета
+- binPrefix - устанавливает префикс для опции bin.
+- noHashSum - отключает хэши кампутных пакетов
```

Comment:
you should take description of this options from the options list Russian page .